addons/editor-compact/userstyle.css

```
@@ -0,0 +1,492 @@
+  border: 0;
+  padding: 0;
+}
+[class*="sound-editor_input-group_"] {
+  gap: 2px;
+}
+[class*="sound-editor_input-group_"] > * {
+  margin: 0;
+}
+[class*="sound-editor_row_"] > [class*="icon-button_container_"] {
+  margin-left: 1rem;
+}
+[class*="sound-editor_button_"],
+[class*="sound-editor_tool-button"],
+[class*="sound-editor_row-reverse_"] > [class*="icon-button_container_"] {
+  width: 24px;
+  max-width: 24px;
+  height: 24px;
+}
+[class*="mode-tools_mode-tools"] {
+  min-height: 0;
+}
+[class*="paint-editor_editor-container-top"] {
+  padding-bottom: 0;
+  border-bottom: none;
+}
+
+[class*="paint-editor_top-align-row"] {
+  padding-top: 0.5rem;
+}
+[class*="paint-editor_mode-selector"] {
+  width: min-content;
+}
+
+[class*="sound-editor_row_"] + [class*="sound-editor_row_"] {
+  margin-top: 0.5rem;
+}
+[class*="sound-editor_waveform-container_"] {
+  margin: 0;
+}
+
+[class*="sound-editor_row-reverse_"] {
+  gap: 2px;
+}
+[class*="sound-editor_row-reverse_"] > [class*="sound-editor_input-group_"] {
+  margin-right: 0.5rem;
+}
+[class*="sound-editor_row-reverse_"] > [class*="icon-button_container_"] {
+  margin: 0;
+}
+
+[class*="paint-editor_canvas-controls"] {
+  height: min-content;
+  margin-top: 0.5rem;
+}
+[class*="asset-panel_detail-area"] [class*="paint-editor_bitmap-button"] {
+  padding: 0 0.25rem;
+}
+[class*="asset-panel_detail-area"] [class*="paint-editor_bitmap-button"] img {
+  padding: 0;
+}
+.sa-onion-controls-container {
+  height: min-content;
+}
+.sa-onion-settings-tip {
+  right: 70px;
+}
+
+/* Modals and prompts */
+[class*="card_header-buttons_"] > div,
+[class*="card_header-buttons-right_"] > div {
+  padding: 0.5rem;
+  margin: 0;
+}
+[class*="card_header-buttons_"] span:not(.sa-debugger-tabs *) {
+  display: none;
+}
+[class*="card_header-buttons_"] > [class*="card_header-buttons-right_"] {
+  padding: 0;
+}
+[class*="card_remove-button_"] > [class*="card_close-icon_"],
+[class*="card_all-button_"] > [class*="card_help-icon_"] {
+  margin: 0;
+}
+.sa-debugger-tabs {
+  padding: 0 0.25rem;
+  margin-left: 0.25rem;
+}
+.sa-debugger-tabs li {
+  padding: 0.25em 1em;
+}
+.sa-debugger-tabs li + li {
+  margin-inline-start: 0.25rem;
+}
+
+[class*="modal_header"],
+[class*="library_filter-bar"] {
+  height: 2rem;
+}
+[class*="close-button_large"] {
+  width: 1rem;
+  height: 1rem;
+}
+
+[class*="library_filter-bar-item"] {
+  height: 1.5rem;
+}
+[class*="filter_filter-icon"],
+[class*="filter_x-icon_"] {
+  width: 0.75rem;
+  margin: 0.25rem 0.5rem !important;
+}
+[class*="filter_filter-input"] {
+  padding-left: 1.75rem !important;
+}
+[class*="library_divider"] {
+  display: none;
+}
+[class*="filter_x-icon-wrapper"] {
+  position: static;
+  margin: 0;
+}
+[class*="library_tag-wrapper"] {
+  height: auto;
+}
+[class*="library_library-scroll-grid"] {
+  height: calc(100% - 2rem - 2rem) !important;
+}
+[class*="modal_header"] + [class*="library_library-scroll-grid"] {
+  height: calc(100% - 2rem) !important;
+}
```

Unnecessary `!important`.
```suggestion
[class*="library_library-scroll-grid"] {
  height: calc(100% - 2rem);
}
[class*="library_library-scroll-grid"][class*="library_withFilterBar"] {
  height: calc(100% - 4rem);
}
```